client/landing/gutenboarding/components/header/style.scss

```diff
@@ -8,7 +8,7 @@
 	align-items: center;
 	justify-content: space-between;
 	border-bottom: 1px solid $light-gray-500;
-	height: $header-height;
```

Comment:
How was this not a sass error? It seems like this variable isn't defined anywhere.

The only case I can find is here:

https://github.com/Automattic/wp-calypso/blob/0d6375eefab2179ca8d73cb1a986b4349c36f7d3/client/components/select-dropdown/style.scss#L4-L8

Which highlights how poorly SASS with its global variables work with a modular setup like this.